.github/workflows/ci.yml

```diff
@@ -88,32 +70,13 @@ jobs:
 
       - run: echo "::add-mask::${{ secrets.LICENSE_SERVER_URL }}"
 
-      - name: Setup .NET SDK (v6.0)
-        uses: actions/setup-dotnet@v1
-        with:
-          dotnet-version: '6.0.x'
-
-      # Unity is about 4GB in size and takes 20 minutes to download and install.
-      # Caching still takes some time, but brings it down to about 5 minutes.
-      - name: Restore cached Unity installation
-        id: cache-unity
-        uses: actions/cache@v2
-        with:
-          path: ${{ matrix.unity-installation-path }}
-          key: ${{ matrix.os }}-${{ matrix.unity-version }}-${{ matrix.unity-version-changeset }}
-
-      - name: Restore previous build
-        uses: actions/cache@v2
-        with:
-          path: samples/artifacts/builds
-          # using hash of package.json to bust cache on release builds which modify it
-          key: samples/artifacts/builds|${{ matrix.os }}-${{ matrix.unity-version }}-${{ matrix.unity-version-changeset }}-${{ hashFiles('package/package.json') }}
-
       - name: Restore Unity Packages
         uses: actions/cache@v2
         with:
-          path: samples/unity-of-bugs/Library
-          key: samples/unity-of-bugs/Library|${{ matrix.os }}-${{ matrix.unity-version }}-${{ matrix.unity-version-changeset }}-${{ hashFiles('samples/unity-of-bugs/Packages/packages-lock.json') }}
+          path: |
+            samples/unity-of-bugs/Library/Packages
+            temp/unity-packages/Library/ScriptAssemblies/*.TestRunner.*
```

Comment:
TestRunner wasn't cached - caching it speeds up (skips) `UnityRestorePackages` target